pkg/minikube/download/preload.go

```diff
@@ -76,6 +77,10 @@ func remoteTarballURL(k8sVersion string) string {
 
 // PreloadExists returns true if there is a preloaded tarball that can be used
 func PreloadExists(k8sVersion, containerRuntime string) bool {
+	if !viper.GetBool("preload") {
+		return false
+	}
+
 	if containerRuntime != "docker" {
```

Comment:
nit: maybe add the issue link for when we will add containerd and crio runtimes.